influxdb3_write/src/write_buffer/metrics.rs

```diff
@@ -4,37 +4,51 @@ use metric::{Metric, Registry, U64Counter};
 
 #[derive(Debug)]
 pub(super) struct WriteMetrics {
-    write_lines_total: Metric<U64Counter>,
-    write_bytes_total: Metric<U64Counter>,
+    write_lines_count: Metric<U64Counter>,
```

Comment:
`write_lines_total` is the more standard naming convention. Metric names should end in units, e.g. _seconds, _bytes, or _total for unit-less metrics. See https://prometheus.io/docs/practices/naming/

Reply:
Addressed this in https://github.com/influxdata/influxdb/pull/25722/commits/cd51bc2beda9a23446d694ee411e409871b1de39